src/commands/filter_parser.cc

```diff
@@ -21,7 +21,6 @@
 #include "absl/strings/numbers.h"
 #include "absl/strings/str_cat.h"
 #include "absl/strings/string_view.h"
-#include "src/valkey_search_options.h"
```

Comment:
This duplicate exists in main. Any fix that is not full text related should be done in main first. I am adding this back